Modules/Scripted/DICOMLib/DICOMWidgets.py

```diff
@@ -81,6 +81,8 @@ class DICOMDetailsBase(VTKObservationMixin, SizePositionSettingsMixin):
   """
 
   widgetType = None
+  loadingFinished = qt.Signal()
```

Comment:
Are these signal used ? If yes, it may be worth documenting both `loadingFinished` and `closed` in the class description